TeddTech/ribby
# chore(sync-branch): Dev

# Checklist

- [x] My code follows the [developer guidelines of this project](https://github.com/TeddTech/ribby/blob/main/.github/CONTRIBUTING.md)
- [x] This pull request has been linted by running `npm run lint-staged` without errors
- [x] This pull request has been tested in **dev** mode by running `npm start` and reviewing affected routes

# What is it for?


# Description

Initial release of blank project template.

# Steps to test

1. Make sure dev sight is running
2. Make sure prod sight is running
3. Make sure security settings isn't blocking any functionality.
